content/applications/sales/point_of_sale/payment/stripe.rst

```diff
@@ -0,0 +1,108 @@
+======
+Stripe
+======
+
+Connecting a payment terminal allows you to offer a fluid payment flow to your customers and ease
+the work of your cashiers.
+
+Configuration
+=============
+
+Configure the payment method
+----------------------------
+
+Start by activating **Stripe** in the settings. From the **POS** dashboard, go to
+:menuselection:`Configuration --> Settings --> Payment Terminals --> Check Stripe`.
+
+Then, create the payment method:
+
+- Click :menuselection:`Configuration --> Payment Methods --> Create`, and complete the
+  :guilabel:`Method` field with your payment method's name;
+- Set the :guilabel:`Journal` field as :guilabel:`Bank` and the :guilabel:`Use a Payment Terminal`
+  field as :guilabel:`Stripe`;
+- Enter your payment terminal serial number in the :guilabel:`Stripe Serial Number` field;
+- Click on :guilabel:`Don't forget to complete Stripe connect before using this payment method.`
+
+.. image:: stripe/create-method-stripe.png
+   :align: center
+   :alt: payment method creation form
+
+.. note::
+   - Do not check the :guilabel:`Identify Customer` field. Doing so would prevent unindentified
+     customers from using Stripe.
+   - The :guilabel:`Outstanding Account` and the :guilabel:`Intermediary Account` can stay empty to
+     use the default accounts.
+   - Find your payment terminal serial number under the device or on `Stripe's website <https://dashboard.stripe.com>`_.
+
+Connect Stripe to Odoo
+----------------------
+
+Click on :guilabel:`Connect Stripe`. Doing so redirects you automatically to a configuration page.
+Fill in all the information to create your Stripe account and link it with Odoo. Once the forms are
+completed, the :guilabel:`Publishable Key` and the :guilabel:`Secret Key` are automatically loaded
+in Odoo, and your terminal is ready to be used in a POS.
+
+.. image:: stripe/stripe-connect.png
+   :align: center
+   :alt: stripe connect form
+
+.. important::
+   - When you use **Stripe** exclusively in Point of Sale, you only need the **Secret Key** to use
+     your terminal.
+   - The :guilabel:`State` is inherent to **eCommerce** and can stay set as :guilabel:`Disabled`.
```

Comment:
I don't think that this formulation conveys what you meant, with "inherent". Also, it is not only related to eCommerce, since payment providers can be used also in customer portals.

I think that it would be best to approach this from the payment provider perspective.

Reply:
I changed it, what do you think?